src/Find.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Find (findMatches, recipeToString) where
```

Comment:
It would be great to have Haddock comments for the exported functions in this file, but it won't hold up merging.